binary/c_parser/testparser.c

```diff
@@ -22,7 +22,7 @@ long rootNode;
 char* vspecfile;
 
 char* getTypeName(nodeTypes_t type) {
-    switch (type) { 
+    switch (type) {
```

Comment:
Shouldn't we have a case for "STRUCT" type nodes and "PROPERTY" type nodes?

Reply:
Good catch! Thanks.